### DISCUSSION TITLE
Touchpad Gestures not working - 3 Finger Middle Tap Click

### DISCUSSION THREAD

**Discussion by vanavata** [rustdesk/rustdesk#6443] [General]
### Bug Description

I updated to the latest nightly build on both host and client side after reading a closed issue about middle click
Re: Multi-button mouse support - https://github.com/rustdesk/rustdesk/issues/3963

In my case I'm using the synaptics touchpad in the HP EliteBook 840G5. 

The 3 finger gesture is configured on the host and client (same laptop and version of Windows 10) but middle tap click doesn't work.

This is an essential feature for me when I'm browsing and clicking links.

Thank you

### How to Reproduce

3 finger tap middle click touchpad gesture doesn't work on remote host when clicking any browser links or an app like notepad in the taskbar (which should open a second instance).

### Expected Behavior

Middle click should work.

### Operating system(s) on local side and remote side

Windows 10 x64 LTSC

### RustDesk Version(s) on local side and remote side

1.2.4 

### Screenshots

![3finger-middle-tapclick](https://github.com/rustdesk/rustdesk/assets/107262451/6f5c43f4-5d2b-4eb7-bc79-dc160d693f18)

**Comment by ayushd03** [comment]
What could be the reason that this still hasnt been fixed ?

**Comment by southerndoc** [comment]
Would also love to have ability to use the three-finger swipe forward/backward on Macs to help navigate websites.

**Comment by pbarney** [comment]
Just curious if a dev has even seen this request or not. A `planned`, `needs triage`, `help wanted` or even `wontfix` tag would be really helpful just for our sake.
